types/rpi-ws281x-native/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for rpi-ws281x-native 1.0
+// Project: https://github.com/beyondscreen/node-rpi-ws281x-native
+// Definitions by: Nathan Rajlich <https://github.com/TooTallNate>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
+
+/// <reference types="node" />
+
+declare enum StripType {
+	SK6812_RGBW = 403703808,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/rpi-ws281x-native/index.d.ts;
  Line: 10;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.